content/about/who-we-are.md

```diff
@@ -34,7 +34,7 @@ Thank you to all our growing list of contributors who have made Quire what it is
 - [@egardner](https://github.com/egardner)
 - [@fchasen](https://github.com/fchasen)
 - [@garrettdashnelson](https://github.com/garrettdashnelson)
-- [@ggarcia0596](https://github.com/ggarcia0596)
+- [@garciagregg](https://github.com/garciagregg)
```

Comment:
This was actually correct before, @ggarcia0596 was one of our MUI interns a few years ago. She must have closed her GitHub account. Not sure if it's best to remove her altogether, or just unlink it. Your call.

Reply:
Ah, ok, I'll just unlink it.